Raspi/Test/mqtt_wireless_log.py

```diff
@@ -0,0 +1,152 @@
+import paho.mqtt.client as mqtt
+import os
+import pandas as pd
+from DataToTempCSV import DataToTempCSV
+import json
+from datetime import datetime
+
+
+def on_connect(client, userdata, flags, rc):
+    """ When the MQTT client connects to the broker it prints out if it
+    connects successfully and then subscribes to all of the wireless-module
+    topics"""
+
+    if rc == 0:
+        print("Connected Successfully! Result code: " + str(rc))
+    else:
+        print("Something went wrong! Result code: " + str(rc))
+
+    # Subscribe to all of the wireless module topics
+    client.subscribe("/v3/wireless-module/#")
+
+
+def on_message(client, userdata, msg):
+    """ MQTT callback for when data is sent on the subscribed
+    '/v3/wireless-module/#' topics. The module_id is found by squashing M
+    infront of the module number. eg M1, M2, M3... etc. The module_id is used
+    for identifying what data came from where and is also used for naming the
+    temp files.
+    """
+
+    module_id = "M" + str(msg.topic[20])
+
+    # Start the recording of <module_id>
+    if msg.topic.endswith("start"):
+        # Clear the old temporary files in the folder, just in case the script
```

Comment:
Debating whether to put the things inside the if statements (apart from the prints) into their own functions. Thoughts?

Reply:
It would probably make it nicer to read but for stop it is literally 2 lines of code.

Reply:
I'll try it and see how it looks. It is a really good point though, I was tempted when I first wrote it.

Reply:
I think it's better as it is. Putting it in a function is another layer of indirection and it's already pretty clear what the purpose of the code block is.